.github/workflows/e2e.yml

```diff
@@ -28,3 +28,26 @@ jobs:
 
     - name: run e2e test
       run: make test-e2e
+  
+  e2e-pr-blocking:
```

Comment:
this should replace the above `make test-e2e` which runs all e2e tests. This change is running all e2e tests + additional run of quick start

Reply:
Yup, make sense.